From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Mon, 22 Aug 2016 17:50:10 +0200
Subject: server.cfg: prepare later M-A substition while built

After enabling dh9 the plugin *.so files will be installed in
/usr/lib/$(DEB_HOST_MULTIARCH). Adding a marker into server.cfg which
can be substituted while package creation.
---
 installer/linux/server.cfg | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/installer/linux/server.cfg b/installer/linux/server.cfg
index 451cf16..06b260e 100644
--- a/installer/linux/server.cfg
+++ b/installer/linux/server.cfg
@@ -340,6 +340,10 @@ user_plugin		= db
 # configuration file of the user plugin, examples can be found in /usr/share/doc/kopano/example-config
 user_plugin_config	= /etc/kopano/ldap.cfg
 
+# location of the kopano plugins
+# In Debian we have Multi-Arch architecture, we look for the plugins in the desired platform folder.
+plugin_path		= /usr/lib/@DEB_HOST_MULTIARCH@/kopano
+
 # scripts which create stores for users from an external source
 # used for ldap and unix plugins only
 createuser_script		=	/etc/kopano/userscripts/createuser
